vessim/cosim/sil_interface.py

```diff
@@ -0,0 +1,164 @@
+from threading import Thread
+from typing import List
+
+from vessim.core.storage import SimpleBattery, DefaultStoragePolicy
+from vessim.cosim._util import VessimSimulator, VessimModel
+from vessim.sil.http_client import HTTPClient, HTTPClientError
+from vessim.sil.node import Node
+from vessim.sil.api_server import VessimApiServer
+from vessim.sil.stoppable_thread import StoppableThread
+
+
+class SilInterfaceSim(VessimSimulator):
+    """Software-in-the-Loop (SiL) interface simulator that executes the model."""
+
+    META = {
+        "type": "time-based",
+        "models": {
+            "SilInterface": {
+                "public": True,
+                "params": [
+                    "nodes",
+                    "battery",
+                    "policy",
+                    "collection_interval",
+                    "api_host",
+                    "api_port"
+                ],
+                "attrs": ["battery", "p_cons", "p_gen", "p_grid", "ci"],
```

Comment:
why is battery a mosaik attribute? wouldn't it make sense to make this simulator a `"any_inputs": True` like the Monitor?